### DISCUSSION TITLE
Table shows empty record values

### DISCUSSION THREAD

**Discussion by hotemetoot** [Budibase/budibase#6458] [Help]
Hi guys, thanks in advance for trying to understand my problem and helping me out =P

I have created three objects:
1. Meeting Types
2. Meeting Notes
3. Tasks

Meeting Notes belong to a single Meeting Type (monthly review, weekly stand-up, etc.)
Tasks belong to a single Meeting Notes (monthly review June 10th)

I have an edit screen for Meeting Notes. This screen also shows two tables:
1. Tasks related to these specific Meeting Notes
2.  Tasks related to the specific Meeting Type related to these Meeting Notes (for examples tasks created during last month's monthly review)

However, something strange is happening in the second table. The Tasks related to the current Meeting Notes show blank field values except for one field (see image below), the last three rows are mostly blank. Any idea what I might be doing wrong?

![image](https://user-images.githubusercontent.com/24481384/175577450-17223d34-6dec-4be5-9e1a-11d3afc65e71.png)

**Comment by melohagan** [comment]
Hi @hotemetoot 

It looks like blank rows have been created somewhere. If you look in the **Data** tab, you should be able to find blank rows that have been saved into that table.

**Reply by hotemetoot** [reply]
Hi @melohagan 

Thank you for the suggestion. I checked in the data tab and it seem like there are no blank rows in there:

![Untitled](https://user-images.githubusercontent.com/24481384/176006508-2010a156-1a77-474a-bc0d-11285f56c6c6.png)

**Reply by melohagan** [reply]
And you tabbed through the pages as well to be sure? Or alternatively you can use the filter in the top right to find where responsible = *is empty*

**Reply by hotemetoot** [reply]
Yes I double checked and filtered on name = empty and due date = empty to be double sure.
What I suspect is that the three empty rows are the same three records from the first table (showing the tasks related to these meeting notes) because the meeting date matches (20-May-22). However, for some reason all of the task fields except for the meeting date show up as blank values.

**Reply by melohagan** [reply]
Are you using this method for displaying the related data? https://docs.budibase.com/docs/fetching-data#displaying-relational-data

**Reply by hotemetoot** [reply]
Yes, although it's a slight variation.

- I have a Meeting Notes details screen (data provider for Meeting Notes with a filter using the url.id to match one specific Meeting Notes record.

- Then I have a second data provider to get the Meeting Type for that specific Meeting Notes record.

- Finally I have a third data provider to get the Tasks related to that Meeting Type.


![image](https://user-images.githubusercontent.com/24481384/176207790-44acb1de-7201-4b65-aba4-46a509a329b0.png)

**Reply by melohagan** [reply]
Thank you for the additional information.

I'm wondering if the fact that you have tasks on both the Meeting Notes and Meeting Types table is causing issues. The Internal Budibase DB is a NoSQL database, and may not handle complex relationships well.

It may be helpful to get a look at your app export if you wanted to send it to support@budibase.com

**Reply by hotemetoot** [reply]
Yes I have no experience with NoSQL so that might be the issue.
I have sent the export file to the support mail with #6458 in the title.
Thank you!

**Reply by hotemetoot** [reply]
Hi @melohagan, did the app export I sent you give you any more information about this issue?

**Reply by melohagan** [reply]
Hey @hotemetoot Sorry for the late response. Would you mind double checking the app export you sent me? I'm not seeing the content from the screenshot in the OP, and there is no Meeting Types table.

I can't figure out where `New Repeater.Meeting Types.Tasks` is coming from.
